Makefile

```diff
@@ -136,6 +141,22 @@ clean:
 	$(RM) -r build
 	$(RM) -r vendor
 
+# File target that generates a diff file if formatting is incorrect
+$(FORMAT_CHECK): $(GO_FILES)
+	@echo "Checking Go file formatting..."
+	@echo $(GO_FILES)
+	mkdir -p $(@D)
+	@$(GOROOT)/bin/gofmt -l -d $(GO_FILES) > $(FORMAT_LOG)
+	@if [ -s $(FORMAT_LOG) ]; then \
+		cat $(FORMAT_LOG); \
+		echo "Formatting issues found. Please run 'gofmt -w .' on the above files and commit the changes."; \
```

Comment:
IMO this message will be confusing since the FORMAT_LOG file includes the diffs too. Better to remove `-d` option from the `gofmt` call so that FORMAT_LOG contains only the file names.

Reply:
Done as suggested